include/zephyr/drivers/video.h

```diff
@@ -753,104 +753,184 @@ struct video_buffer *video_buffer_alloc(size_t size);
+ * @endverbatim
+ */
+#define VIDEO_PIX_FMT_GBRG8      VIDEO_FOURCC('G', 'B', 'R', 'G')
+#define VIDEO_PIX_FMT_GBRG8_BITS 8
+
+/**
+ * @verbatim
+ * | Gggggggg | Rrrrrrrr | Gggggggg | Rrrrrrrr | Gggggggg | Rrrrrrrr | ...
+ * | Bbbbbbbb | Gggggggg | Bbbbbbbb | Gggggggg | Bbbbbbbb | Gggggggg | ...
+ * @endverbatim
+ */
+#define VIDEO_PIX_FMT_GRBG8      VIDEO_FOURCC('G', 'R', 'B', 'G')
+#define VIDEO_PIX_FMT_GRBG8_BITS 8
+
+/**
+ * @verbatim
+ * | Rrrrrrrr | Gggggggg | Rrrrrrrr | Gggggggg | Rrrrrrrr | Gggggggg | ...
+ * | Gggggggg | Bbbbbbbb | Gggggggg | Bbbbbbbb | Gggggggg | Bbbbbbbb | ...
+ * @endverbatim
+ */
+#define VIDEO_PIX_FMT_RGGB8      VIDEO_FOURCC('R', 'G', 'G', 'B')
+#define VIDEO_PIX_FMT_RGGB8_BITS 8
 
 /**
  * @}
  */
 
 /**
  * @name RGB formats
+ * Per-color (R, G, B) channels.
  * @{
  */
 
-/** RGB565 pixel format */
-#define VIDEO_PIX_FMT_RGB565 video_fourcc('R', 'G', 'B', 'P') /* 16  RGB-5-6-5 */
+/**
+ * 5-bit blue followed by 6-bit green followed by 5-bit red, in little-endian over two bytes.
+ * @verbatim
+ * | gggRrrrr | BbbbbGgg | ...
+ * @endverbatim
+ */
+#define VIDEO_PIX_FMT_RGB565      VIDEO_FOURCC('R', 'G', 'B', 'P')
+#define VIDEO_PIX_FMT_RGB565_BITS 16
 
-/** XRGB32 pixel format */
-#define VIDEO_PIX_FMT_XRGB32 video_fourcc('B', 'X', '2', '4') /* 32  XRGB-8-8-8-8 */
+/**
+ * There is an empty (X) byte for each pixel.
+ * @verbatim
+ * | Xxxxxxxx Rrrrrrrr Gggggggg Bbbbbbbb | ...
+ * @endverbatim
+ */
+#define VIDEO_PIX_FMT_XRGB32      VIDEO_FOURCC('B', 'X', '2', '4')
+#define VIDEO_PIX_FMT_XRGB32_BITS 32
 
 /**
  * @}
  */
 
 /**
  * @name YUV formats
+ * Luminance (Y) and chrominance (U, V) channels.
  * @{
  */
 
-/** YUYV pixel format */
-#define VIDEO_PIX_FMT_YUYV video_fourcc('Y', 'U', 'Y', 'V') /* 16  Y0-Cb0 Y1-Cr0 */
+/**
+ * There is either a missing channel per pixel, U or V.
+ * The value is to be averaged over 2 pixels to get the value of individual pixel.
+ * @verbatim
+ * | Yyyyyyyy Uuuuuuuu | Yyyyyyyy Vvvvvvvv | ...
+ * @endverbatim
+ */
+#define VIDEO_PIX_FMT_YUYV      VIDEO_FOURCC('Y', 'U', 'Y', 'V')
+#define VIDEO_PIX_FMT_YUYV_BITS 16
 
-/** XYUV32 pixel format */
-#define VIDEO_PIX_FMT_XYUV32 video_fourcc('X', 'Y', 'U', 'V') /* 32  XYUV-8-8-8-8 */
+/**
+ * There is an empty (X) byte for each pixel.
+ * @verbatim
+ * | Xxxxxxxx Yyyyyyyy Uuuuuuuu Vvvvvvvv | ...
+ * @endverbatim
+ */
+#define VIDEO_PIX_FMT_XYUV32      VIDEO_FOURCC('X', 'Y', 'U', 'V')
+#define VIDEO_PIX_FMT_XYUV32_BITS 32
 
 /**
- *
  * @}
  */
 
 /**
- * @name JPEG formats
+ * @name Compressed formats
  * @{
  */
 
-/** JPEG pixel format */
-#define VIDEO_PIX_FMT_JPEG video_fourcc('J', 'P', 'E', 'G') /*  8  JPEG */
-
 /**
- * @}
+ * Both JPEG (single frame) and Motion-JPEG (MJPEG, multiple JPEG frames concatenated)
  */
+#define VIDEO_PIX_FMT_JPEG VIDEO_FOURCC('J', 'P', 'E', 'G')
 
 /**
  * @}
  */
 
 /**
- * @brief Get number of bytes per pixel of a pixel format
+ * @brief Get number of bit per pixel of a pixel format
```

Comment:
nits: number of bits

Reply:
`variable umber` typo also spotted below...
Thank you for the proofreading!